packages/application-shell/src/test-utils/test-utils.spec.js

```diff
@@ -264,3 +274,117 @@ describe('router', () => {
     expect(history.location.pathname).toBe('/foo');
   });
 });
+
+describe('custom render function', () => {
+  describe('wrapper', () => {
+    describe('when provided', () => {
+      const Context = React.createContext();
+      const ProvidedWrapper = ({ children }) => (
+        <Context.Provider value="provided wrapper">{children}</Context.Provider>
+      );
+      ProvidedWrapper.propTypes = {
+        children: PropTypes.node.isRequired,
+      };
+
+      it('should renderApp merges the provided wrapper with its own wrapper', () => {
```

Comment:
This sentence seems broked 😅. 

```suggestion
      it('should merge the passed wrapper with renderApp\'s internal wrapper', () => {
```